### ISSUE TITLE
Fresh Checkout Doesn't build with BrowserslistError

### ISSUE THREAD

**Issue by danrossi** [videojs/video.js#8253]
There is a build error with the output

```


> video.js@8.3.0 build:js
> rollup -c


src/js/index.js → dist/video.js...
[!] (plugin babel) BrowserslistError: Unknown version 2 of kaios
src/js/index.js
```

**Comment by philjhale** [comment]
@danrossi In order to help we will need more detailed steps to replicate the problem. Are you using Mac or Windows? Exactly what commands are you executing.

If I clone the video.js repository and execute the following commands, I'm not seeing any errors. 
```
npm install
npm run build
```

**Comment by danrossi** [comment]
Fresh checkout. npm install and npm run build. Let me try again.

**Comment by danrossi** [comment]
I'm sorry I thought I found a legitimate bug. I had to checkout twice and its building now.
